src/math.rs

```diff
@@ -252,8 +252,8 @@ impl<S: BaseFloat> Isometry3<S> {
 
 #[derive(Debug, Clone)]
 pub struct Obb<S> {
-    pub isometry: Isometry3<S>,
-    pub isometry_inv: Isometry3<S>,
+    pub from_obb: Isometry3<S>,
```

Comment:
maybe here a comment like in the struct OrientedBeam would help understand the meaning of the isometry

Reply:
I renamed all transformations, ptal!